pkgs/tools/X11/xidlehook/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib, rustPlatform, fetchFromGitHub, x11, xorg, libpulseaudio, pkgconfig, makeWrapper }:
+
+rustPlatform.buildRustPackage rec {
+  name = "xidlehook-${version}";
+  version = "0.4.5";
+
+  src = fetchFromGitHub {
+    owner = "jD91mZM2";
+    repo = "xidlehook";
+    rev = version;
+
+    sha256 = "0mj02w959h0sr8b2r3l54dqn4j3y88xdr434y3jyfqd371kg0i39";
+  };
+
+  cargoSha256 = "0grx7qcyah6xclkl944a4cg4fvmma7k6qnsakwzanwqcv8snkbzg";
```

Comment:
also for this one:
```
directory = "/build/source/vendor"
fixed-output derivation produced path '/nix/store/nsn04g4mflzdjsddz9hc8578jfbspj9y-xidlehook-0.4.5-vendor' with sha256 hash '0a1bl6fnfw6xy71q3b5zij52p9skylj1ivqj8my44bfsid2qfn7d' instead of the expected hash '0grx7qcyah6xclkl944a4cg4fvmma7k6qnsakwzanwqcv8snkbzg'
```